version/generate_version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# generates the version of file based on current git version
+
+cd $GOPATH/src/github.com/contiv/netplugin/version
+
+USE_RELEASE=$1
+
+if [ -z "${USE_RELEASE}" ]
+then
+  file_name=version_default.go
+  UseVersionStr="// +build !use_version" 
+  useDefaultVersionString="False"
+else
+  file_name=version_release.go
+  UseVersionStr="// +build use_version" 
+  useDefaultVersionString="True"
+fi
+
+GitCommitStr=`git show | grep commit | cut -f 2 -d ' ' | cut -b 1-10`
+VersionStr=`cat CURRENT_VERSION`
+BuildTimeStr=`date`
+
+cat > $file_name << EOF
+$UseVersionStr
+// NOTE: this file is auto-generated by PR commits
+
+package version
+
+const usedDefaultVersion = "$useDefaultVersionString"
+
+// sample values are specified here
+const (
```

Comment:
oh I just meant define a var, no need for consts. Defining this var eliminates need for `Get()` as anyways we are just generating one version, we don;t need a way to instantiate it multiple times.

Also note that we really don't need fields in Info struct to be public so we can make them lower case and avoid anyone trying to access them directly. 

```
var MyVersion = Info {
  gitCommit:  "$GitCommitStr"
  version: "$VersionStr"
  buildTime: "$BuildTimeStr"
}
```